### DISCUSSION TITLE
[firebase_database] Flutter Web compatibility

### DISCUSSION THREAD

**Discussion by loic-hamdi** [firebase/flutterfire#6689] [Q&A]
When can we please expect the release of Firebase Realtime Database with a Flutter Web compatibility ?

[https://github.com/FirebaseExtended/flutterfire#firebase_database](https://github.com/FirebaseExtended/flutterfire#firebase_database)

**Comment by deepak786** [comment]
@zzterrozz PR https://github.com/FirebaseExtended/flutterfire/pull/6952 is already in Progress for web support.

**Comment by loic-hamdi** [comment]
@deepak786 thank you for the update. What is the estimated delivery date?
